### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by markshuttle**
## Update the README to reflect Kubernetes operators

Rework the README with links to the new Juju website.

## QA steps

None needed, doc change only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,131 @@
...
+The OLM provides a central mechanism for operator instantiation,
+configuration, upgrades, integration and administration. The OLM provides a
+range of [operator lifecycle services](https://juju.is/operator-services)
+including leader election and persistent state. Instead of manually
+deploying and configuring operators, the OLM manages all the operators in a
+model at the direction of the administrator.
+
+## Open Operator Collection
+
+A world's [largest collection of operators](https://charmhub.io/) all use
```
Comment (wallyworld) [inline README.md:56]: s/A/The ?

File: README.md [hunk 2]
```diff
@@ -1,61 +1,131 @@
...
-
-
- - **Maintain portability and repeatability**  
- Retain control over your deployment and eliminate the need for vendor-specific offerings.
- Your devops tooling should be cloud-agnostic and Kubernetes-aware.
-
+[![Juju logo](doc/juju-logo.png?raw=true)](https://juju.is/)
+
+[Juju is a model-driven **Operator Lifecycle Manager**
+(OLM)](https://juju.is/overview). Juju greatly improves the experience of
```
Comment (wallyworld) [inline README.md:4]: Perhaps

Juju is a model-driven Operator Lifecycle Manager (OLM), which greatly improves ...

ie avoid 2 consecutive sentences starting with "Juju".

File: README.md [hunk 3]
```diff
@@ -1,61 +1,131 @@
...
+A world's [largest collection of operators](https://charmhub.io/) all use
+Juju as their OLM. The Charmhub community emphasizes quality, collaboration
+and consistency. Publish your own operator and share integration code for
+other operators to connect to your application.
+
+The [Open Operator Manifesto](https://charmhub.io/manifesto) outlines the
+values of the community and describe the ideal behaviour of operators, to
+shape contributions and discussions.
+
+## Multi cloud and hybrid operations across ARM and x86 infrastructure
```
Comment (wallyworld) [inline README.md:65]: There's a lot of up front Kubernetes text before we reach this section, which seem (to me) to belatedly mention that Juju works in other places as well. Did we want to balance the intro a bit and then dive into the benefits on k8s?

**Review by wallyworld — COMMENTED** [review]
Thanks for the nice improvements.
A couple of minor suggestions, and also a more substantial question.

File: README.md (hunk 1 above)
Comment (markshuttle) [reply README.md:56]: Yes, good catch

File: README.md (hunk 2 above)
Comment (markshuttle) [reply README.md:4]: Yes, but I am also hopeful that the more pedantic approach is better for "what is..." type search queries.

File: README.md (hunk 3 above)
Comment (markshuttle) [reply README.md:65]: Let's emphasize K8s for the moment.

**Review by wallyworld — APPROVED** [review]

**Review by jameinel — APPROVED** [review]

**Pull request merged**